src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -600,13 +600,15 @@ Copyright (c) .NET Foundation. All rights reserved.
   ============================================================
   -->
 
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' and '$(_IsExecutable)' == 'true'">
-    <ProjectCapability Include="CrossPlatformExecutable" />
-  </ItemGroup>
-
-  <!-- Reference Manager capabilities -->
+  <!-- NETCoreApp capabilities -->
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
     <ProjectCapability Remove="ReferenceManagerAssemblies" />
+    <ProjectCapability Remove="ReferenceManagerCOM" Condition="$(_TargetFrameworkVersionWithoutV) &lt; '3.0'" />
+    <ProjectCapability Include="CrossPlatformExecutable" Condition="'$(_IsExecutable)' == 'true'" />
+  </ItemGroup>
```

Comment:
@davkean Should these be moved to VS targets as in https://github.com/dotnet/project-system/pull/4400/files

We still don't seem to have clear guidance on when to put here and when to put there.